app/scenes/Schedule/index.js

```diff
@@ -15,7 +15,7 @@ import moment from 'moment';
 
 import type { ScheduleTalk } from '../../types';
 
-import Splash from 'react-native-smart-splash-screen';
+// import Splash from 'react-native-smart-splash-screen';
```

Comment:
Yeah we don't have great support for a splashscreen that animates into the app for various reasons. It works meh on iOS right now, didn't bother tweaking it further because the image loader will always potentially cause a flicker. Either way, I'll keep the fork updated at https://expo.io/@community/reactconf2017 :)